### PR TITLE
Main site

### PR DESCRIPTION
**Pull request by norbertbrzeski**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,8 +93,110 @@
         </div>
     </div>
     <!--The End of Functionalities-->
+    <div class="cov zdj1"></div>
```
Comment (MarcinFortuna) [inline index.html]: No Polish class names please

File: stylesheet.css [hunk 2]
```diff
@@ -514,3 +516,146 @@ table {
...
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: cover;
+    background-attachment: fixed;
+    height: 80vh;
+
+}
+
+.zdj1 {
+    background-image: url("mainautobusy.jpg");
```
Comment (MarcinFortuna) [inline stylesheet.css]: put these files to Assets

**Review by MarcinFortuna — CHANGES_REQUESTED** [review]
Looks good, but there are some small issues (see inline comments)

**Comment by piotrbarcik** [comment]
Looks good, change small details about Marcin writes and will be OK.

**Review by piotrbarcik — APPROVED** [review]
👎

**Review by piotrbarcik — CHANGES_REQUESTED** [review]
👎

**Pull request closed without merging**